students/TressaHood/lesson02/fizz_buzz.py

```diff
@@ -0,0 +1,26 @@
+#lesson 2, assignment fizz buzz
+
+#define the main function
+def main():
+
+	#print 1 - 100 inclusive
+	for i in range(1,101):
```

Comment:
You'll want 4 space indentions for Python.  Recommend removing the extra blank lines within your function as well.

Reply:
It's weird because in my tab preferences it says 4? not sure why this is.